Makefile

```diff
@@ -12,7 +12,17 @@ dep_install:
 install:
 	@cd $(PROJECT_PATH) && dep ensure && go install
 
-run: install
+es:
```

Comment:
I believe that `es` is not the best name for docker compose because it should run cerebro too

Reply:
Ok, it's just for ES... I think that we can use a docker fo this project later